### PR TITLE
[W6.4d][W10-B2]Ong Kuan Yang

### PR DESCRIPTION
**Pull request by ongkuanyang**
Adds isMutating method.

### CODE REVIEW

File: src/seedu/addressbook/logic/Logic.java [hunk 1]
```diff
@@ -85,7 +85,9 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
         command.setData(addressBook, lastShownList);
         CommandResult result = command.execute();
-        storage.save(addressBook);
```
Comment (okkhoy) [inline src/seedu/addressbook/logic/Logic.java:88]: Need to update the header comment of execute method to reflect the new behavior of save.

**Review by okkhoy — COMMENTED** [review]
What happens to logic test?

**Comment by okkhoy** [comment]
@ongkuanyang 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/logic/Logic.java (hunk 1 above)
Comment (ongkuanyang) [reply src/seedu/addressbook/logic/Logic.java:88]: Done.

**Comment by ongkuanyang** [comment]
Updated LogicTest. Closing PR.

**Pull request closed without merging**